extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/FastBootHibernatePersistenceProvider.java

```diff
@@ -410,6 +414,16 @@ private static void injectDataSource(String persistenceUnitName, String dataSour
             return;
         }
 
+        if (multiTenancyStrategy != null && (multiTenancyStrategy == MultiTenancyStrategy.DATABASE
+                || multiTenancyStrategy == MultiTenancyStrategy.SCHEMA)) {
```

Comment:
```suggestion
        if (multiTenancyStrategy != null && multiTenancyStrategy != MultiTenancyStrategy.NONE
                && multiTenancyStrategy != MultiTenancyStrategy.DISCRIMINATOR) {
```

not that this would make a difference 😃, but maybe let's keep the condition the same as in the other place so it's easier to spot this "counterpart"?